src/Pagination/components/PageNumberButton.ts

```diff
@@ -1,22 +1,52 @@
-import { SFC, createElement } from "react";
+import { ReactElement, createElement } from "react";
 import * as classNames from "classnames";
 
 interface PageNumberButtonProps {
     pageNumber: number;
     selectedPageNumber: number;
-    onClickAction: (pageNumber: number) => void;
+    onClickAction: () => void;
     key?: string | number;
 }
 
-export const PageNumberButton: SFC<PageNumberButtonProps> = (props: PageNumberButtonProps) => createElement("li", {
-        className: classNames(
-            props.selectedPageNumber === props.pageNumber ? "active" : "",
-            props.pageNumber < 10 ? "single-digit" : ""
-        ),
-        onClick: () => props.onClickAction(props.pageNumber),
-        key: props.key
-    },
-    props.pageNumber
-);
+export const PageNumberButton = (
+    props: PageNumberButtonProps
+): ReactElement => {
+    const { onClickAction, pageNumber, selectedPageNumber } = props;
+
+    return createElement(
+        "li",
+        {
+            className: classNames(
+                selectedPageNumber === pageNumber ? "active" : ""
+            ),
+            role: "button",
+            onClick: onClickAction,
+            onKeyDown: onKeyDown.bind(null, onClickAction),
```

Comment:
If you use `key` instead of `keyCode` at line 39 you can use just `onKeyDown`

Reply:
What do you mean exactly?

Reply:
I only use keydown now